src/Page/Pokemons/Pokemons.tsx

```diff
@@ -103,35 +130,41 @@ const Pokemons: FC = () => {
   return (
     <>
       <TableContainer sx={{ mt: 8 }} component={Paper}>
-        <Table sx={{ minWidth: 650 }} aria-label="simple table">
-          <TableHead>
-            <MultipleSelectChip
-              setPersonName={types => {
-                setSearchParams(prev => ({
-                  ...prev,
-                  rowsPerPage: Number(searchParams.get('rowsPerPage') ?? 10),
-                  page: 0,
-                  types: qs.stringify(types),
-                }));
-              }}
-              personName={
-                Object.values(
-                  qs.parse(searchParams.get('types') ?? ''),
-                ) as string[]
-              }
-            />
+        <MultipleSelectChip
+          setPokemonTypes={types => {
+            setSearchParams(prev => ({
+              ...prev,
+              rowsPerPage: Number(searchParams.get('rowsPerPage') ?? 10),
+              page: 0,
+              types: qs.stringify(types),
+            }));
+          }}
+          pokemonTypes={
+            Object.values(qs.parse(searchParams.get('types') ?? '')) as string[]
```

Comment:
а может тут лучше так?
```suggestion
            Object.values(qs.parse(searchParams.get('types') ?? [])) as string[]
```

и тогда у тебя в нижних компонентах уйдет куча проверок. так как у тебя будет в любом случае маасив, хотя бы пустой